proto/nym/vpn.proto

```diff
@@ -95,6 +95,29 @@ message ImportUserCredentialRequest {
 
 message ImportUserCredentialResponse {
   bool success = 1;
+  ImportError error = 2;
+}
+
+message ImportError {
+  enum ImportErrorType {
+    IMPORT_ERROR_TYPE_UNSPECIFIED = 0;
+    VPN_RUNNING = 1;
+    CREDENTIAL_ALREADY_IMPORTED = 2;
+    GENERIC = 3;
+    STORAGE_ERROR = 4;
+    DESERIALIZATION_FAILURE = 5;
+    CREDENTIAL_EXPIRED = 6;
+    VERIFICATION_FAILED = 7;
+    FAILED_TO_QUERY_CONTRACT = 8;
+  }
+
+  ImportErrorType kind = 1;
+
+  // Detailed error message for logging and debugging
+  string message = 2;
+
+  // Optional additional details
+  map<string, string> details = 3;
```

Comment:
use the `optional` attribute for optional field

Reply:
`optional` is not supported on `map` fields, and it's already effectively optional since an empty map should be serialized to zero bytes